src/main/target/BLUEJAYF4/target.h.orig

```diff
@@ -0,0 +1,175 @@
+/*
```

Comment:
What's this `.orig` file for? Presumably it's a leftover from the merge and should be deleted?